README.md

```diff
@@ -2,9 +2,86 @@
+|                   | Symbol            | &#9989;  |                     |
+|                   | Table             | &#9989;  |                     |
+|                   | Tank              | &#10060; | Add later (date unknown, low priority) |
+|                   | Text Symbol       | &#10060; | Add later           |
+|                   | Text Update       | &#9989;  |                     | 
+|                   | Thermometer       | &#10060; | Add later (date unknown, low priority) |
+| Controls          | Action Button     | &#9989;  |                     |
+|                   | Boolean Button    | &#9989;  |                     |
+|                   | Check Box         | &#9989;  |                     |
+|                   | Choice Button     | &#9989;  |                     |
+|                   | Combo Box         | &#9989;  |                     |
+|                   | File Selector     | &#10060; | Add later           |
+|                   | Radio Button      | &#9989;  |                     |
+|                   | Scaled Slider     | &#9989;  |                     |
+|                   | Scrollbar         | &#10060; | Add later           |
+|                   | Slide Button      | &#10060; | Add later (date unknown, low priority) |
+|                   | Spinner           | &#10060; | Add later (date unknown, low priority) |
+|                   | Text Entry        | &#9989;  |                     |
+|                   | Thumbwheel        | &#10060; | Add later (date unknown, low priority) |
+| Plots             | Data Browser      | &#10060; | Add later (date unknown, low priority) |
+|                   | Image             | &#10060; | Unknown             |
+|                   | Stripchart        | &#10060; | Unknown             |
+|                   | XY Plot           | &#9989;  |                     |
+| Structure         | Array             | &#10060; | Add later           |
+|                   | Embedded Display  | &#9989;  |                     |
+|                   | Group             | &#9989;  |                     |
+|                   | Navigation Tabs   | &#9989;  |                     |
+|                   | Tabs              | &#9989;  |                     |
+|                   | Template/Instance | &#10060; | Unknown             |
+
+#### Features
+
+| Widget      | Included | Reason                  |
+| :---------: | :------: | :---------------------: |
+| Actions     | &#9989;  | Some actions may not be supported. Please open an issue for any issues noticed. |
+| Formulas    | &#10060; | `sim://` PVs are supported, but not `eq://`. This will be added in future. |
+| Rules       | &#9989;  | Partial support. x, y and Font rules are currently not supported. This will be added in future. Please open an issue for any issues noticed.|
+| Scripts     | &#10060; | The use of scripting is recommended against in general by CSS Developers. Formulae should be able to handle most use cases.  |
```

Comment:
Maybe expand CSS to CS-Studio so people don't get confused with CSS web styles?

Reply:
Ah good idea, thanks